src/pages/CommunityResourcesPage/ui/CommunityResourcesPage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import ResourceCard from "../../../shared/ui/ResourceCard/ResourceCard";
+import { resources } from "./constants";
+import styles from "./CommunityResourcesPage.module.css";
+import Button from "../../Pomodoro/components/Button/Button";
```

Comment:
Reusable Button component should be created and maintained in shared/ui folder.
You can use this one for now, but in the future when we will have custom buttons (ticket #33 https://github.com/catbytes-community/webplatform-frontend/issues/33) you should switch to using custom button

Reply:
Okay)